src/environments/environments/util.py

```diff
@@ -82,16 +61,6 @@ def reduce_lidar(lidar: LaserScan):
 
         return new_range
 
-def reduce_lidar_n(lidar: LaserScan, num_points: int):
```

Comment:
Same as above. Would rather merge this and normal reduce lidar.

Reply:
So you'd prefer if reduce_lidar_n was the saved function? So everytime we call it we define how many points we'd like?

Reply:
Yeahh imo – except we'd name it reduce lidar. They both have the exact same implementation but reduce_lidar is hard coded 10 and the other one isn't

Reply:
Fixed in latest commit